### PR TITLE
Clarify early proposal phases

### PR DESCRIPTION
**Pull request by taralx**
This attempts to clarify the early proposal phases, especially the difference between phase 0 and phase 1. I have attempted to normalize the format of the text in the various phases as well.

### CODE REVIEW

**Comment by taralx** [comment]
@binji as promised 🙂

File: process/phases.md [hunk 1]
```diff
@@ -21,52 +21,47 @@ In general, the process moves forward through a series of numbered phases.
...
 
-During this phase the new repository forking the spec repo is populated. Pull requests and issues are used to iterate on the design of the feature. If a high level descriptions in a design repo style .md files is useful,
-these are added to describe the feature as it's being designed.
+During this phase:
 
-If relevant to demonstrate the viability of a feature, prototype
-implementations of the feature are implemented in interested embedders (possibly
-on a branch).
-The Community Group will attempt to reach broad consensus.
+  1. A [tracking issue](https://github.com/WebAssembly/design/labels/tracking) is created and added to the [list of future features](https://github.com/WebAssembly/design/blob/master/FutureFeatures.md#tracking-issues).
```
Comment (binji) [inline process/phases.md:47]: We should discuss whether this should be phased out. It seems the folks prefer using the proposal repo for this instead.

Comment (tlively) [reply process/phases.md:47]: +1 to phasing this out. I didn't know we still had tracking issues 😅

Comment (rossberg) [reply process/phases.md:47]: +1. The tracking issues didn't have a lot of utility, AFAICT. At least I never looked at them (or created them :) ), the proposal repo is much more convenient.

**Comment by taralx** [comment]
Updated per CG to remove the tracking issue stuff.

**Comment by taralx** [comment]
@dtig @binji I think one of you has to push the merge button?

**Comment by binji** [comment]
Yep, I was waiting to see if anyone else wanted to comment. But it's been 7 days, I imagine this is good to land now (and we can adjust later anyway).

**Pull request merged**